### PR TITLE
Missing menu entries when going to Jobs, Events #167

### PR DESCRIPTION
**Pull request by evalica**
* Fixing build: "A file was included in `/_layouts/default.html` that is a symlink or does not exist in your `_includes` directory"
* No links are displayed, so adding a local global.yml

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -5,7 +5,9 @@
                 <i class="fa fa-bars"></i>
             </button>
             <a class="navbar-brand page-scroll" href="/#page-top">
-                <span class="logo">{% include img/opensourcedesign-logo.svg %}</span>
+                <span class="logo">
+                    <img src="//opensourcedesign.net/images/opensourcedesign-logo.svg" alt="Open Source Design Logo"/>
```
Comment (jancborchardt) [inline _includes/header.html]: We don’t need this alt text, as the logo is purely cosmetic. It says "Open Source Design" in the next line, so we can use `alt=""` here. :)

Comment (evalica) [reply _includes/header.html]: I don't know about the 'need', but it's a standard/accessibility practice to provide Alts for every image.

Comment (jancborchardt) [reply _includes/header.html]: Yes it is, but it's also accessibility practice to not duplicate text, and to use _empty_ alt text for these images. :) (Alt tag present but being empty like `alt=""`

See:

> Use an empty alt attribute for any image that is decorative or not necessary for understanding the content of the page (alt="" or just alt - e.g. ). If, for example, the information in a graph is explained in text below it, there’s no need to duplicate that in alt text.

From https://a11yproject.com/posts/alt-text/

Comment (evalica) [reply _includes/header.html]: k :) First I though you wanted to remove it, but indeed you mentioned it to be empty. 
According to https://www.w3.org/TR/WCAG20-TECHS/H67.html I will make this change. Thanks Jan.

Comment (evalica) [reply _includes/header.html]: Created similar change for the JOBS repo at https://github.com/opensourcedesign/jobs/pull/320/files

**Review by elioqoshi — APPROVED** [review]

**Comment by elioqoshi** [comment]
Good stuff, merged!

**Pull request merged**